src/current/_includes/v24.1/app/retry-errors.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-Your application should [use a retry loop to handle transaction errors]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#transaction-retry-errors) that can occur under [contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).
+When running under the default [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}) isolation level, your application should [use a retry loop to handle transaction errors]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#transaction-retry-errors) that can occur under [contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention). Client-side retry handling is **not** necessary under [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) isolation.
```

Comment:
Is this the best link for `SERIALIZABLE`? I expected it to be a reference.

Reply:
Yeah, we lack a dedicated "reference" doc since `SERIALIZABLE` was synonymous with "CockroachDB" until recently. The closest would be the Transaction Layer doc, but that one is too low-level to be an intro.